docs/workflow/testing/coreclr/unix-test-instructions.md

```diff
@@ -40,6 +40,35 @@ The `src/tests/build.sh` script generates the Core_Root folder, which contains t
 
 The output will be at `<repo_root>/artifacts/tests/coreclr/<os>.<arch>.<configuration>/Tests/Core_Root`.
 
+## Building Test Subsets
+
+The `src/tests/build.sh` script supports three options that let you limit the set of tests to build;
+when none of these is specified, the entire test tree (under `src/tests`) gets built but that can be
+lengthy (especially in `-priority1` mode) and unnecessary when working on a particular test.
+
+1) `-test:<test-project>` - build a particular test project specified by its project file path,
+either absolute or relative to `src/tests`. The option can be specified multiple times on the command
```

Comment:
Please mention that the option that they could provide multiple items and separate them by semicolon.

Reply:
Updated in 9th commit.